PyISY/Nodes/node.py

```diff
@@ -157,22 +157,8 @@ def dim(self):
             return True
 
     def _get_notes(self):
-        # Get the device notes, currently only the Spoken property is saved.
```

Comment:
The code to parse this XML could probably be made into a module level function and then called by both Node and Group. That way we don't have to duplicate code.